app/controllers/cloud_volume_controller.rb

```diff
@@ -129,6 +129,22 @@ def cloud_volume_form_fields
     }
   end
 
+  def storage_manager_form_fields
+    assert_privileges("cloud_volume_new")
+    ems = find_by_id_filtered(ExtManagementSystem, params[:id])
+
+    form_fields = { :emstype => ems.emstype }
+
+    case ems.type
+    when "ManageIQ::Providers::Amazon::StorageManager::Ebs"
```

Comment:
Is this ok to do a check like this in the UI? Please note that I expect further form fields for Amazon, but not for Openstack (for now, because also the existing code only allows cloud tenant).